src/components/card-link/card-link.module.css

```
@@ -0,0 +1,34 @@
+/*
+NOTE: `color: black` is needed because the color is otherwise set by a global
+style applied to anchor elements.
+*/
+.root {
+  color: black;
+
+  /* Only enable animation if query is supported and no-preference on motion */
+  @media (prefers-reduced-motion: no-preference) {
+    transition: background-color 0.2s;
```

This would make it ✨ fancy
```suggestion
    transition: background-color 0.2s, box-shadow 0.2s;
```

---

Agree!! Didn't think to add this in after adding the hover state changes. 😊